GetIntoTeachingApiTests/Jobs/LocationBatchJobTests.cs

```diff
@@ -45,9 +46,10 @@ public async void RunAsync_InsertsNewLocations()
             await _job.RunAsync(JsonConvert.SerializeObject(batch));
             await _job.RunAsync(JsonConvert.SerializeObject(batch));
 
-            DbContext.Locations.Count().Should().Be(batch.Count());
+            DbContext.Locations.Count().Should().Be(batch.Count);
             DbContext.Locations.ToList().All(l =>
                 batch.Any(b => BatchLocationMatchesExistingLocation(b, l))).Should().BeTrue();
+            DbContext.Locations.All(l => l.Source == Source.CSV);

```

Comment:
We've covered the scenario for when a new location gets the correct CSV source, but we're missing a test for the scenario where an existing location exists with Unknown source and is then updated to CSV - highlighted by SonarCloud here:

https://sonarcloud.io/component_measures?id=DFE-Digital_get-into-teaching-api&metric=new_coverage&pullRequest=395&selected=DFE-Digital_get-into-teaching-api%3AGetIntoTeachingApi%2FJobs%2FLocationBatchJob.cs&view=list